pysap/plugins/mri/reconstruct/fourier.py

```diff
@@ -120,27 +120,49 @@ def adj_op(self, x):
         return pfft.ifft2(self._mask * x)
 
 
-class NFFT2(FourierBase):
-    """ Standard 2D non catesian Fast Fourrier Transform class
+class NFFT(FourierBase):
+    """ ND non catesian Fast Fourrier Transform class
+    The NFFT will normalize in a symmetric way the direct and adjoint operator.
```

Comment:
I think the specific type of normalisation used should be described in this docstring somwhere.

Reply:
the real problem s that the FFT and the NFFT doesn't take the same convention and this is weird for the same package. so we do have the choice, keep the unitary convention which will define the FFT as a orthogonal bases or keep the original convention of the FFT

Reply:
Do you have any preference for which convention should be used?